proper_paranthetics.py

```diff
@@ -0,0 +1,27 @@
+def fn(s):
+    """
+    shows the state of parenthasies
+    if
+    """
+    s=list(s)
+    s.reverse()
+    n=0
+    while s:
+        t = s.pop()
+        if t == "(": 
+            n += 1
+        elif t == ")": 
+            n -= 1
+
+        if n < 0:
```

Comment:
I get why you are checking if its greater than zero though. Good job on dealing with the edge cases.